apps/app.go

```diff
@@ -113,19 +123,26 @@ type Asset struct {
 	Key    string    `json:"key"`
 }
 
+// Manifest describes a "known", installable app. They generally come from the
+// marketplace, and can also be installed as overrides by developers.
+// Manifest should be abbreviated as `m`.
 type Manifest struct {
-	AppID       AppID      `json:"app_id"`
-	Type        AppType    `json:"app_type"`
-	Version     AppVersion `json:"version"`
-	DisplayName string     `json:"display_name,omitempty"`
-	Description string     `json:"description,omitempty"`
+	AppID   AppID      `json:"app_id"`
+	Type    AppType    `json:"app_type"`
+	Version AppVersion `json:"version"`
 
+	DisplayName string `json:"display_name,omitempty"`
+	Description string `json:"description,omitempty"`
 	HomepageURL string `json:"homepage_url,omitempty"`
 
-	// HTTPRootURL applicable For AppTypeHTTP.
-	//
-	// TODO: check if it is used in the // user-agent, consider naming
-	// consistently.
+	OnDisable   *Call `json:"on_disable,omitempty"`
```

Comment:
On Enable/Disable should come as soon as we add `apps disable/enable` command :)

Reply:
Is `enable/disable` p1 feature? I thought we where only having `install/uninstall`.

Reply:
I had the flag in my branch, already used in a few places (like `/apps list`) - figured no harm leaving it in the code even if not implemented yet, seems a rather obvious next step?